app/controllers/uploads_controller.rb

```diff
@@ -4,14 +4,14 @@ def new
   end
 
   def create
-    @upload = Upload.new(upload_params)
+    @folder = Folder.find(params[:upload][:folder_id])
+    @upload = @folder.uploads.new(upload_params)
```

Comment:
@AELSchauer the fact we saw that image in the PR after implemented the file upload, means that it was working, at least locally. Yes, let's discuss.